src/renderer/icons/BigSpinner/index.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+
+import React from "react";
+import png from "./bigspinner.png";
+
+type Props = {
+  size?: number,
+};
+
+function BigSpinner({ size }: Props) {
+  const defaultedSize = size || 44;
+  return <img style={{ width: defaultedSize, height: defaultedSize }} src={png} alt="" />;
```

Comment:
Why is this a PNG now instead of an SVG? 🤔

Reply:
because we never made the SVG working for the gradient. we spent an hour on this and it's broken.

Reply:
oh, okay!